6-figures-Surefire

```diff
@@ -0,0 +1,4 @@
+I am going to have multiple 6 figures job.
+Everything is possible 
+
```

Comment:
please remove this gap

Reply:
It's very easy to resolve

Reply:
the gap is really big

Reply:
It will make it lighter and simple

Reply:
it true you have to remote this gap. thanks

Reply:
i think the gap makes sens

Reply:
he plans to insert one information there.